pkg/api/v1/types.go

```diff
@@ -582,6 +582,8 @@ type RBDVolumeSource struct {
 	// Defaults to false.
 	// More info: http://releases.k8s.io/HEAD/examples/rbd/README.md#how-to-use-it
 	ReadOnly bool `json:"readOnly,omitempty"`
+	// Optional: Sidecar is the sidecar container's name
+	Sidecar string `json:"sidecar,omitempty"`
```

Comment:
Neither the name of this field nor the comment helps me understand what this is or what its intended usage is. Is this a container or an image? If a container, is it Docker's name for the container, or something else? What are you trying to do? 

Why is this exposed by the API at all? My interpretation of #13138 is that you wanted a container-based plugin mechanism. Kubelet plugins shouldn't be exposed via the user-facing API.